src/vtcomposite.cpp

```diff
@@ -583,61 +583,89 @@ struct InternationalizeWorker : Napi::AsyncWorker
 {
     using Base = Napi::AsyncWorker;
 
-    using property_list = std::vector<std::pair<std::string, vtzero::data_view>>; // maybe this exists in vtzero
-
     InternationalizeWorker(std::unique_ptr<InternationalizeBatonType>&& baton_data, Napi::Function& cb)
         : Base(cb),
           baton_data_{std::move(baton_data)},
           output_buffer_{std::make_unique<std::string>()} {}
 
-    std::vector<std::string> processWorldview(const std::string& pval) const
+    // for a given feature, determine how many clones of the feature are
+    // required given a worldview.
+    // - If the feature has _mbx_worldview: all, return {"all"}
+    // - If there is no requested worldview "null", determine which legacy worldviews should be created
+    std::vector<std::string> worldviews_for_feature(std::string const& pval) const
     {
-        static const std::vector<std::string> legacy_worldviews{"CN", "IN", "JP", "US"};
-
         if (pval == "all")
         {
             return {"all"};
         }
 
+        // convert pval into vector of strings US,CN => {"US", "CN"}
+        // and sort ascending
+        std::vector<std::string> worldview_values;
+        std::stringstream s_stream(pval); // create string stream from the string
```

Comment:
nit: avoiding comments that don't add info

```suggestion
        std::stringstream s_stream(pval);
```